My_work_3/Simple_task_2_plots.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+D = np.array([])
+with open("Data.txt", "r") as file:
```

Comment:
np.loadtxt or np.fromfile, right? Best practices tell us "Don't invent a new bicycle in case you already have one and know how it works". Cause in case of more complex data format manual parsing may cause a lot of AGONIZING PAIN. :). Sometimes it is even more convenient to use pandas to import your data so that you can make something like:

df = pandas.readcsv(filename, optins...)
plot(df['first_named_column'])
df['first_named_column'] + 2* df['second_column']

where column names may replace the column numbers in case you have corresponding header line in your file and set proper options to reader routine. It is even possible to make this work with numpy structured dataset.

Reply:
I'll search info about these functions.